Assets/MRTK/Core/Utilities/Editor/ScriptedImporterAssetReimporter.cs

```diff
@@ -43,7 +43,7 @@ public static void OnPostprocessAllAssets(string[] importedAssets, string[] dele
 
                         if (numAttempts <= 3)
                         {
-                            Debug.LogWarning($"Asset '{asset}' appears to have failed importing, will attempt to re-import. Attempt: {numAttempts}");
+                            Debug.Log($"Asset '{asset}' appears to have failed importing, will attempt to re-import. Attempt: {numAttempts}");
```

Comment:
I wonder if we can go a step further and _only_ log if the import fails `numAttempts` times? We don't really need to know that it's trying again...just that it failed

Reply:
good call. done.